core/src/main/java/org/infinispan/conflict/impl/DefaultConflictManager.java

```diff
@@ -446,28 +450,26 @@ public boolean tryAdvance(Consumer<? super Map<Address, CacheEntry<K, V>>> actio
             if (nextSegment < totalSegments) {
                try {
                   if (trace)
-                     log.tracef("Attempting to receive all replicas for segment %s with topology %s", nextSegment, topology);
+                     log.tracef("Cache %s Attempting to receive all replicas for segment %s with topology %s", cacheName, nextSegment, topology);
                   segmentRequestFuture = stateReceiver.getAllReplicasForSegment(nextSegment, topology);
                   long remainingTime = timeService.remainingTime(endTime, TimeUnit.MILLISECONDS);
                   List<Map<Address, CacheEntry<K, V>>> segmentEntries = segmentRequestFuture.get(remainingTime, TimeUnit.MILLISECONDS);
```

Comment:
@danberindei Yes, I wanted to point to this blocking invocation, but it seemed to me as out-of-scope of the PR. Maybe it's time to replace spliterators with reactive.